src/App/Service/User/Exception/UserNotFound.php

```diff
@@ -12,6 +13,11 @@ public static function fromEmail(string $email) : self
         return new self(sprintf('User with email "%s" was not found', $email));
     }
 
+    public static function fromId(UuidInterface $uuid) : self
+    {
+        return new self(sprintf('User with ID "%s" was not found', (string)$uuid));
```

Comment:
I take it back, `__toString` makes it possible: https://3v4l.org/Pm82B
but still, no harm in it :)